test/util/testnode/config.go

```diff
@@ -124,8 +124,7 @@ func DefaultConfig() *Config {
 		WithAppConfig(DefaultAppConfig()).
 		WithAppOptions(DefaultAppOptions()).
 		WithAppCreator(cmd.NewAppServer).
-		WithSuppressLogs(true).
-		WithConsensusParams(DefaultConsensusParams())
```

Comment:
Why remove this?

Reply:
due to working on this branch for too long, I don't remember, but added them back